### PR TITLE
[58] select status of equipment

### PR DESCRIPTION
**Pull request by oaedward**
# PR Description
## Description

- By default, Equipment Status is **Active Deployed**
- Other available lists describing equipment status are in the dropdown

## Related Issue

#58 
## Screenshots (if appropriate):
1. Select Status of Equipment: 

Screenshot 2023-02-17 at 16 10 18

### CODE REVIEW

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-ASSET/ASSET-MaintenanceAndSupport.vue [hunk 1]
```diff
@@ -92,16 +84,14 @@
...
-const status = inject("status");
-const userManual = inject("userManual");
-const technicalManual = inject("technicalManual");
+const MaintenanceAndSupport = inject("MaintenanceAndSupport");
 
 const statusList = ref([
-  "Operational",
-  "Decommissioned",
-  "Requires Maintenance",
+  { name: "Active in Storage", value: "Active in Storage" },
```
[the remark this answers is not in the document]
Comment (oaedward) [reply src/client/web/src/components/BIOMD/BIOMD/BIOMD-ASSET/ASSET-MaintenanceAndSupport.vue:90]: One is for the name stored in the database and the other is what the user sees

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-ASSET/ASSET.vue [hunk 2]
```diff
@@ -237,11 +243,11 @@ export default {
     provide("district", district);
     provide("location", location);
 
-    provide("supportTeam", supportTeam);
-    provide("vendorId", vendorId);
-    provide("status", status);
-    provide("userManual", userManual);
-    provide("technicalManual", technicalManual);
+    // provide("supportTeam", supportTeam);
```
[the remark this answers is not in the document]
Comment (oaedward) [reply src/client/web/src/components/BIOMD/BIOMD/BIOMD-ASSET/ASSET.vue:246]: Definetely

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY-FacilityInformation.vue [hunk 3]
```diff
@@ -108,7 +108,7 @@
         </div>
 
         <!-- GPS Coordinates -->
-        <div class="col-lg-6">
+        <!-- <div class="col-lg-6">
```
[the remark this answers is not in the document]
Comment (oaedward) [reply src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY-FacilityInformation.vue:111]: Alright

**Review by akshaypuli — APPROVED** [review]

**Review by irushavm — APPROVED** [review]
:shipit:

**Pull request merged**